integration/spark-common/src/main/scala/org/apache/carbondata/spark/load/DataLoadProcessBuilderOnSpark.scala

```diff
@@ -172,6 +342,7 @@ object DataLoadProcessBuilderOnSpark {
     updateLoadStatus(model, partialSuccessAccum)
   }
 
+
```

Comment:
revert this change

Reply:
This is WIP PR, I will update you once ready.
I am making 3 flows. one of load, one of insert with converter, insert without converter. I have duplicated code for this. Once all the test case is fixed. I will extract common code from all 3 flows.